examples/ScatterPlot/main.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftPlot
+import AGGRenderer
+import SVGRenderer
+
+var filePath = "examples/Reference/"
+let fileName = "_19_scatter_plot"
+
+var x = [Float]()
+var y = [Float]()
+
+var x1 = [Float]()
+var y1 = [Float]()
+
+for index in 1...100 {
+    x.append(Float(index))
+    y.append(Float(index*2) + Float.random(in: -10...10))
+}
+
+for index in 1...100 {
+    x1.append(Float(index))
+    y1.append(Float(index) + Float.random(in: -10...10))
+}
+
+var agg_renderer: AGGRenderer = AGGRenderer()
+var svg_renderer: SVGRenderer = SVGRenderer()
+
+var plotTitle: PlotTitle = PlotTitle()
```

Comment:
Unless you need to be explicit about a type, I usually tend to leave the type declaration off of code like this. Also, if all you're going to do later is set the title, I might change this to something like:

```
let plotTitle = PlotTitle(title:"SCATTER PLOT")
```

or even do the initialization inline with setting `scatterPlot.plotTitle = ...`.